releasenotes/notes/backendv1-d0d0642ed38fed3c.yaml

```diff
@@ -0,0 +1,9 @@
+---
+deprecations_providers:
+  - |
+    The ``BackendV1`` class is deprecated and it will be removed not earlier than the next major release.
+    There are several migration paths available depending on the main purpose of the backend object:
+
+      - To expose backend information with no access to execution (just a hardware description), consider constructing a :class:`.Target` directly.
+      - To provides access to execution capabilities, consider relying on the primitives interfaces instead.
+      - Alternatively, to continue providing simultaneous :class:`.Target` (hardware information) and ``run`` (execution) capabilities, consider moving to :class:`.BackendV2` (see https://qisk.it/backendV1-to-V2).
```

Comment:
I just realized, the reno is missing the `qiskit.providers.models` deprecation.

Reply:
good point! added in baa16f14eea93d56b8eaf09b468a02309d2c0549

Reply:
I think the commit wasn't pushed